README.md

```diff
@@ -87,6 +87,14 @@ source .env
 
 1. Link to parachain [dashboard](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A11946)
 
+1. Off-boarding MRC from relay chain
+
+    ```bash
+    ./scripts/init.sh offboard-mrc
```

Comment:
@saraswatpuneet is the offboard script expected to open up the paraId for reuse?

I tried it and while it offboarded the id 2000, when running register, it chose 2001 instead.

Reply:
@wilwade  no, para_id once registered is taken, this ensures we safely offboard mrc and onboarding it again for same para_id but with updated genesis/wasm with new changes we push in. This in part is only safe to do in devenet as in testnet and betanet we would be doing runtime upgrades etc.

Reply:
This will prevent us from always having to register new para_ids but that option is also on the table

Reply:
Ah I see. So it doesn't require re-running the register step, only the onboard step.

Reply:
yes, we will only run register step when we re-deploy relay for any reason so register mrc step is tied to relay deployment